### PR TITLE
docs: GST E Invoicing

### PR DESCRIPTION
**Pull request by nextchamp-saqib**
Docs for https://github.com/frappe/erpnext/pull/23455

<details>
<summary>
Preview
</summary>
<br>
Setup E Invoice Docs
</details>

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/regional/india/setup-e-invoicing.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<!-- add-breadcrumbs -->
+# E Invoicing under GST
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/regional/india/setup-e-invoicing.md:2]: @nextchamp-saqib 

There's no context... please incorporate answers to the following questions in the documentation

Why was e invoicing introduced?
what is GSP? what is IRN? full form? uses? Why would someone wanna cancel IRN?

What is relevance of eway bill in einvoicing?

File: erpnext_documentation/www/docs/user/manual/en/regional/india/setup-e-invoicing.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+1. **GSTIN**: GSTIN by which your company is registered on e-invoice portal.
+1. **Username**: Username created in previous step.
+1. **Password**: Password created in previous step.
+
+### Generating IRN
+
+Create a sales invoice and keep it under Draft state. Click on **E Invoicing** button group and then on **Generate IRN**. If Sales Invoice doesn't have any validation errors, IRN will be generated and updated in the Sales Invoice. You can now submit the invoice and print E Invoice with QRCode image by selecting **GST E Invoice** Print Format while printing.
+
+<img class="screenshot" src="/docs/assets/img/regional/india/einv_gen_irn_button.png">
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/regional/india/setup-e-invoicing.md:41]: 3 images with no context, add sentences in between saying what the images are about

File: erpnext_documentation/www/docs/user/manual/en/regional/india/setup-e-invoicing.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+<!-- add-breadcrumbs -->
+# E Invoicing under GST
+
+Under GST, all small and large businesses are supposed to register all sales invoices onto GST portal that falls under one of the following categories:
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/regional/india/setup-e-invoicing.md:4]: all small and large businesses? isnt' it only for companies with turnover above 500 crore?

Comment (nextchamp-saqib) [reply erpnext_documentation/www/docs/user/manual/en/regional/india/setup-e-invoicing.md]: Its mandatory for 100Cr+ businesses from 1st Jan. Also its going to be the same for all businesses from 1st April 2021.

Right now I have changed this to 100Cr+

File: erpnext_documentation/www/docs/user/manual/en/regional/india/setup-e-invoicing.md (hunk 2 above)
Comment (nextchamp-saqib) [reply erpnext_documentation/www/docs/user/manual/en/regional/india/setup-e-invoicing.md:38]: Done

File: erpnext_documentation/www/docs/user/manual/en/regional/india/setup-e-invoicing.md (hunk 1 above)
Comment (nextchamp-saqib) [reply erpnext_documentation/www/docs/user/manual/en/regional/india/setup-e-invoicing.md:2]: Done! Let me know whats missing now.

File: erpnext_documentation/www/docs/user/manual/en/accounts/contents.md [hunk 4]
```diff
@@ -104,6 +104,7 @@ You can check out [this video](https://www.youtube.com/watch?v=5wjollWN0OA) for
 1. [GST Reminders](/docs/user/manual/en/regional/india/gst-reminders)
 1. [GST 3B](/docs/user/manual/en/regional/india/gst-3b-report)
 1. [Auto generating e-Way Bill](/docs/user/manual/en/regional/india/auto-generate-e-way-bill-JSON)
+1. [Setup E Invoicing](/docs/user/manual/en/regional/india/setup-e-invoicing)
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/accounts/contents.md:107]:
```suggestion
1. [Setup E-Invoicing](/docs/user/manual/en/regional/india/setup-e-invoicing)
```

File: erpnext_documentation/www/docs/user/manual/en/regional/india/setup-e-invoicing.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+Once you cancel the IRN, the invoice will look something like this.
+
+<img class="screenshot" src="/docs/assets/img/regional/india/einv_cancelled_irn.png">
+
+## 3. E-Way bill
+
+### 3.1 Generating E-Way Bill
+
+E-Way Bill can also be generated along with IRN if **Transporter Info** is provided. You can find the Transporter Info section in the bottom part of the invoice. You must select **Transporter**, **Mode of Transport**, and **Distance** to generate E-Way Bill. You can also generate E Way Bill after generating IRN and submitting the invoice.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/regional/india/setup-e-invoicing.md:64]: what is the link between E-Way bill and E-Invoicing?

**Review by prasadarr — APPROVED** [review]

**Pull request merged**